markdown/dev/reference/api/path/join/en.md

```diff
@@ -8,9 +8,12 @@ The `Path.join()` method joins this path with another path.
 ## Signature
 
 ```js
-Path path.join(path other)
+Path path.join(path other[, bool closed=false])
```

Comment:
Same remark. No need for the square brackets as the optional value has a default assigned.